src/transforms/groupby.js

```diff
@@ -39,11 +39,11 @@ exports.attributes = {
         valType: 'string',
         description: [
             'Pattern by which grouped traces are named. If only one trace is present,',
-            'defaults to the group name (`"%g"`), otherwise defaults to the group name',
-            'with trace name (`"%g (%t)"`). Available escape sequences are `%g`, which',
-            'inserts the group name, and `%t`, which inserts the trace name. If grouping',
+            'defaults to the group name (`"%{group}"`), otherwise defaults to the group name',
+            'with trace name (`"%{group} (%{trace})"`). Available escape sequences are `%{group}`, which',
```

Comment:
Interesting. The only tricky part is that to be really precise, it would have to be `%{transforms[2].group}`, which it then links up with the schema to realize the pluralized key is in the schema and is a data array, and can be indexed and expanded to `%{transforms[2].groups[12]}`. The problem I see with that is that templateString must then receive the schema and an item index and that the key must contain the transform number. Correct me if I'm mistaken though.